blog/content/post/2025-11-19-path-to-hsi3.md

```diff
@@ -0,0 +1,596 @@
+---
+title: 'The Dasharo Path to HSI-3'
+abstract: 'Dasharo on MeteorLake NovaCustom laptops has reached HSI-3.
+           This took extending support for Intel BootGuard in coreboot and
+           combining TPM event logs of coreboot and EDK.'
+cover: /covers/hsi-3.svg
+author: sergii.dmytruk
+layout: post
+published: true    # if ready or needs local-preview, change to: true
+date: 2025-11-19
+archives: "2025"
+
+tags:
+  - firmware
+  - dasharo
+  - coreboot
+  - edk2
+  - fwupd
+  - tpm
+categories:
+  - Firmware
+  - Security
+
+---
+
+An important milestone has been achieved by Dasharo firmware on September 18
+with the publication of NovaCustom releases that pass [level 3][hsi-3] checks of
+[Host Security ID][hsi].  This post provides background information on what it
+means and explains what went into making this happen.  At the moment of
+publication this information is relevant only to [V540TU] and [V560TU] models
+(no dGPU, only integrated graphics).  Models with dGPU (V540TNx and V560TNx)
+had their firmware release postponed due to technical issues but everything
+said here will apply to them in the near future.
+
+[hsi-3]: https://fwupd.github.io/libfwupdplugin/hsi.html#hsi3-protected-state
+[hsi]: https://fwupd.github.io/libfwupdplugin/hsi.html
+
+[V540TU]: https://shop.3mdeb.com/product/v54-series-14-dasharo-laptop-by-novacustom/
+[V560TU]: https://shop.3mdeb.com/product/v56-series-16-dasharo-laptop-by-novacustom/
+
+## What is Host Security ID (HSI)
+
+HSI is essentially an attempt to come up with a metric of how secure a given
+system is.  Evaluating HSI score is implemented in [fwupd][fwupd] (which itself
```

Comment:
```suggestion
system is. Evaluating HSI score is implemented in [fwupd][fwupd] (which itself
```

Reply:
OK, didn't expect this rabbit hole to be so deep. We are always learning something new.